source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -121,6 +121,10 @@ def isDescendantOf(self, obj: "NVDAObjects.NVDAObject") -> bool:
 		return bool(res)
 
 	def _get_positionInfo(self):
+		if self.role == controlTypes.Role.TABLECELL:
+			for anc in reversed(api.getFocusAncestors()):

```

Comment:
This is definitely going to return wrong information when the table cell doesn't have focus, such as when using object navigation

I think you want to traverse the parents of the objects instead.